DESCRIPTION

```diff
@@ -54,17 +54,16 @@ Imports:
     tidyr,
     utils
 Suggests:
-    arm,
     bayesplot,
     caret,
     glmnetUtils,
     knitr,
     mbart,
-    nnls,
```

Comment:
At some point, we could consider dropping mia dependency. The IntegratedLearnerFeomMAE is using some functionality that are related to input testing and data wrangling.

However, we could implement them in IntegratedLearner directly as it does not require too much work and adding mia increases the dependency list.

Reply:
Create an issue on this